docs/access_token-pop.md

```diff
@@ -0,0 +1,99 @@
+# (Preview) MSAL Android: Requesting Proof of Possession Protected Access Tokens
+
+## What is a PoP/AT?
+To understand the purpose and benefits offered by PoP, it is important to understand the security posture of a standard Bearer token.
+
+A standard access token from Azure Active Directory (AAD) is a Bearer token in JWT format corresponding to [RFC-7523](https://tools.ietf.org/html/rfc7523). These tokens can be used by anyone possessing the token to access the audience (`aud`) described in the JWT. Because a Bearer token may be used by anyone in possession of it, it is possible that malicious actors may replay tokens that have been leaked by resources or intercepted over the wire from break-and-inspect proxies against a resource to access protected systems or data.
+
+Proof of Possession (PoP) increases the security posture of these tokens embedding them inside of a JWT envelope and signing (binding) that JWT with RSA key material. The key material is generated on the device which was originally issued the tokens and never leaves it. The resulting JWT is called the Signed HTTP Request (SHR). This binding renders the access token unusable without a recent proof for the target resource endpoint.
+
+## How long is a PoP/AT valid?
+By default, Bearer access tokens issued by AAD have a 1 hour validity. The validity of the Signed HTTP Request may be shorter, depending on the resource middleware configuration.
+
+When an SHR is created by the client, a timestamp (`ts`) claim is embedded in the JWT. The resource middleware will, upon receipt of the token, inspect its signature and timestamp to ensure integrity (anti-tamper protection) and validity (non-expiry). By default, the SAL middleware honors a 5 minute validity period meaning that once an SHR been signed by the client it may be used for 5 minutes before the resource will require a newly signed access token.
```

Comment:
Do you think if we should call out that the one we expose in AcquireTokenResult is not going to match this 5 minute value (and they shouldn't rely on that - instead, they should always be calling MSAL to get a newly-minted token instead of caching it)

Reply:
Good call out -- I'll add a little blurb on this

Reply:
Blurb added in afbe5c921f19b9900eba49a4b6e8cd7553b368f2